### PR TITLE
ASRMAINT-979 Add CustomerNumber field to Customer

### PR DESCRIPTION
**Pull request by troygizzi**

### CODE REVIEW

File: OpenTrack.nuspec [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
     <id>OpenTrack.NET</id>
-    <version>2.2.0</version>
+    <version>2.2.1</version>
     <title>OpenTrack.NET</title>
     <tags>OpenTrack, DealerTrack, DMS</tags>
     <description>This library works as an interface between the OpenTrack API from DealerTrack's DMS and a utilizing system.</description>
     <projectUrl>http://github.com/ASRPro/OpenTrack.NET</projectUrl>
 	<licenseUrl>https://github.com/ASRPro/OpenTrack.NET/blob/master/LICENSE.md</licenseUrl>
     <authors>ASR Pro LLC, Matt Wagner, Brian French</authors>
-    <owners>ASR Pro LLC, Matt Wagner</owners>
```
Comment (mattgwagner) [inline OpenTrack.nuspec]: :-1:  haha

Comment (superkarn) [inline OpenTrack.nuspec]: You're still listed as one of the authors.  Just not owner any more :sunglasses:

Comment (superkarn) [inline OpenTrack.nuspec]: @troygizzi please fix this conflict.  See https://github.com/asrpro/OpenTrack.NET/commit/0f8d19345539ab96b781d192bd837871b8443f0e

**Comment by troygizzi** [comment]
These changes are covered by PR 12 and PR 15.

**Pull request closed without merging**